website/source/api/system/replication.html.md

```diff
@@ -197,7 +197,7 @@ identifier can later be used to revoke a secondary's access.
 
 | Method   | Path                         | Produces               |
 | :------- | :--------------------------- | :--------------------- |
-| `GET`    | `/sys/replication/primary/secondary-token` | `200 application/json` |
+| `POST`   | `/sys/replication/primary/secondary-token` | `200 application/json` |
```

Comment:
This (and elsewhere) should probably be updated to sys/replication/performance/...